configure.ac

```diff
@@ -196,6 +196,23 @@ then
     AM_CFLAGS="$AM_CFLAGS -DWOLFTPM_SWTPM"
 fi
 
+# Windows TBS device Support
+AC_ARG_ENABLE([winapi],
+    [AS_HELP_STRING([--enable-winapi],[Enable use of TPM through Windows driver (default: disabled)])],
+    [ ENABLED_WINAPI=$enableval ],
+    [ ENABLED_WINAPI=no ]
+    )
+
+if test "x$ENABLED_WINAPI" = "xyes"
+then
+    if test "x$ENABLED_DEVTPM" = "xyes" -o "x$ENABLED_SWTPM" = "xyes"
+    then
+        AC_MSG_ERROR([Cannot enable swtpm or devtpm with windows API])
+    fi
+
+    AM_CFLAGS="$AM_CFLAGS -DWOLFTPM_WINAPI -DNO_WOLFSSL_SERVER -DNO_WOLFSSL_CLIENT"
```

Comment:
@dgarske I think it will be cleaner to disable the tls examples for `--enable-winapi` in this PR and have a follow on PR. I have a lot of it done, but I was running into permission issues testing with real hardware and realize it may be better to handle as a separate PR.

Reply:
@dgarske  is this an open topic? :)

Reply:
Yes this will be addressed in a new / separate PR that Jeff is already working on here https://github.com/elms/wolfTPM/tree/mingw/tls